.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
In many cases, this `insert_final_newline = true` expected  by github, so better to have it automated.